bokeh/plotting/tests/test_helpers.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import absolute_import
+
+import pytest
+
+import bokeh.plotting.helpers as ph
```

Comment:
most test files test lots of functions in a single module under test, so I like to import the module in question with a short name, and use that throughout, rather than try to do individual imports from the module in question.